lib/easypost/exceptions/invalid_parameter_error.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'easypost/constants'
+
+class EasyPost::Exceptions::InvalidParameterError < EasyPost::Exceptions::EasyPostError
+  def initialize(parameter, tip = nil)
+    super EasyPost::Constants::INVALID_PARAMETER % parameter + (tip.nil? ? '' : tip)
+  end
```

Comment:
suggestion: `tip` isn't immediately understandable here. I had to think for awhile before realizing a synonym might be `suggestion`? If so, I'd maybe even add a comment as to what the suggestion param is for. Do we use it?

Reply:
Resolved